safegraph_py_functions/cbg_functions.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+
+Original file is located at
+    https://colab.research.google.com/drive/1VGPpzYlMPzeVjmuuIH2g7rIC90qMI7lq
+"""
+
+import pandas as pd
+import json
+import os
+import numpy
+import glob
+from zipfile import ZipFile
+from functools import partial
+from multiprocessing import Pool
+
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+from google.colab import auth
+from oauth2client.client import GoogleCredentials
+
+def get_cbg_field_descriptions(year=2019):
+
+  auth.authenticate_user()  # Authenticate and create the PyDrive client. 
+  gauth = GoogleAuth()
+  gauth.credentials = GoogleCredentials.get_application_default()
+  drive = GoogleDrive(gauth)
+
+  def get_drive_id(filename): #function to pull input files from Google Drive
+      drive_ids = {'2016' : '13dLXo67IZDh3OZl042GQYG16Qn_NB7sz',
+                  '2017' : '1b2RVBDgzdrDJkL0OCYRMGBc5zbLhv5MB',
+                  '2018' : '1r7z3efdS5viIRMsQzu9ExHoIL29QjaVi',
+                  '2019' : '1fJsLm6voxWsTq5FQrUzO9PpBltQ8n_lJ'
+                  }
+      return(drive_ids[filename])
+
+  def pd_read_csv_drive(id, drive, dtype=None): #function to pull input files from Google Drive into pandas dataframes
+    downloaded = drive.CreateFile({'id':id})
+    downloaded.GetContentFile('Filename.csv')  
+    return(pd.read_csv('Filename.csv',dtype=dtype))
```

Comment:
This is re-defined in the next function. Can you pull it out so it's only defined once?